packages/cashier/src/components/cashier-container/real/real.tsx

```diff
@@ -1,23 +1,47 @@
 import React from 'react';
 import { Loading } from '@deriv/components';
+import ArrowBack from '../../arrow-back';
+import './real.scss';
 
 type TRealProps = {
+    clearIframe: VoidFunction;
     iframe_height: number | string;
     iframe_url: string;
-    clearIframe: VoidFunction;
+    is_deposit?: boolean;
+    is_eu?: boolean;
     is_loading: boolean;
+    onMountDeposit?: () => Promise<void>;
+    setIsDeposit?: (value: boolean) => void;
 };
 
-const Real = ({ iframe_height, iframe_url, clearIframe, is_loading }: TRealProps) => {
+const Real = ({
+    clearIframe,
+    iframe_height,
+    iframe_url,
+    is_deposit,
+    is_eu,
+    is_loading,
+    onMountDeposit,
+    setIsDeposit,
+}: TRealProps) => {
+    const should_show_arrow_back = !is_eu && is_deposit && Boolean(iframe_height);
+    const should_show_loader = is_loading || !iframe_height;
+
     React.useEffect(() => {
         return () => {
             clearIframe();
+            onMountDeposit?.();
```

Comment:
When the user clicks on `arrow_back` button, the `Real` component will be unmounted, `iframe_url` will be destroyed and `CashierOnboarding` page will be loaded. In order to recreate new `iframe_url` we should run `onMountDeposit` method, that if the user wants to click on `Deposit via bank wire, credit card, and e-wallet ` section again, the deposit page will be loaded with the new `iframe_url`.